File: musos/app/models/song.rb
# == Schema Information
#
# Table name: songs
#
#  id         :integer          not null, primary key
#  name       :text
#  date       :datetime
#  project_id :integer
#  song_notes :text
#  created_at :datetime         not null
#  updated_at :datetime         not null
#

class Song < ActiveRecord::Base
  belongs_to :project
  
end

File: musos/app/assets/javascripts/models/Song.js
var app = app || {};

app.Song = Backbone.Model.extend({
  urlRoot: "/songs",

  defaults: {
    name: "",
    date: "",
    project_id: "",
    song_notes: ""
  },

  initialize: function () {
    console.log("A new song was created");
  }
})
